src/detect-tls-certs.c

```diff
@@ -148,7 +149,7 @@ static InspectionBuffer *TlsCertsGetData(DetectEngineThreadCtx *det_ctx,
     const SSLState *ssl_state = (SSLState *)f->alstate;
     const SSLStateConnp *connp;
 
-    if (f->flags & STREAM_TOSERVER) {
+    if (cbdata->flags & STREAM_TOSERVER) {
```

Comment:
Questions:
1. Why is direction a `TlsCertsGetDataArgs` specific thing?
2. Why don't we just pass around a direction or `flags` as fn args as done in other places? Is `cbdata->flags` supposed to store something else too?

Reply:
the API doesn't have a direction arg, and most users of the API don't need it.